### PR TITLE
Format type UI

### PR DESCRIPTION
**Pull request by otrok7**
Group format types in the Admin UI.  The format types are expandable and configurable.  Some format types can be represented as radio buttons, and the system validates that one is selected-

### CODE REVIEW

File: src/app/Repositories/FormatRepository.php
Comment (jbraswell) [inline src/app/Repositories/FormatRepository.php:90]: Did you mean to pass $lang here?

File: src/tests/Unit/FormatTypesTest.php
Comment (jbraswell) [inline src/tests/Unit/FormatTypesTest.php:16]: Should this file be removed? Or are you still working on it?

File: src/tests/Feature/GetServiceBodiesTest.php
Comment (jbraswell) [inline src/tests/Feature/GetServiceBodiesTest.php:62]: Why did this change?

**Review by jbraswell — CHANGES_REQUESTED** [review]
Aside from the comments I left, my only real request is to choose a name for the new column other than "api key". I understand why you chose that name, but I find it confusing, because an "api key" is usually a type of authentication credentials for an API.

After those small changes, I'm okay with merging this.

File: docker/sample-bmlt-schema.sql
Comment (jbraswell) [inline docker/sample-bmlt-schema.sql:17]: I don't think this schema file should be modified. The new database migrations should take over and create the new table.

**Comment by otrok7** [comment]
Thanks for the review, and for catching the bug.  I changed api_key to api_enum, hope that is OK.  Can you have a look at why the tests are not running.  Locally, everything is fine, and the message is not helpful.  It says the file name doesn't match with the class name, but it does.  I need another set of eyes on this.

**Comment by jbraswell** [comment]
> Thanks for the review, and for catching the bug.  I changed api_key to api_enum, hope that is OK.  Can you have a look at why the tests are not running.  Locally, everything is fine, and the message is not helpful.  It says the file name doesn't match with the class name, but it does.  I need another set of eyes on this.

The name api_enum seems fine, and sure, I'll take a look at the tests. Might be a day or two - busy weekend.

**Comment by pjaudiomv** [comment]
somethings causing phpunit to crash from the last commit maybe in this diff https://github.com/bmlt-enabled/bmlt-root-server/compare/45abec92064fdd25883c230f56f6089b71d3055f...e9dfe714acf42cb1e16786b916fc491d5507c3a8 error messaging is less than ideal but think the file name error is a red herring

**Review by jbraswell — APPROVED** [review]

**Review by pjaudiomv — COMMENTED** [review]
can we update the changelog

**Review by jbraswell — APPROVED** [review]

**Review by pjaudiomv — APPROVED** [review]

**Pull request merged**
